pkg/volume/csi/vol_data.json

```diff
@@ -0,0 +1 @@
+{"driverName":"test-driver","volumeHandle":"test-vol1"}
```

Comment:
why this change?

Reply:
will remove perhaps created as part of test. Dont see it on master